### DISCUSSION TITLE
Have this error: An error has occurred. See the log file Null

### DISCUSSION THREAD

**Discussion by sstanich** [dbeaver/dbeaver#19244] [Q&A]
Hi all, I have dbeaver installed from windows store version and today started to give me this error: An error has occurred. See the log file Null

Any Idea on how to solve this? Also is there any, if I can not run again this app to recover the config I have for all conections?

**Comment by LonwoLonwo** [comment]
Could you please provide the log files? You can find instructions here: https://dbeaver.com/docs/wiki/Log-files/. Thank you.

You can back up your workspace and point to it to restore your configuration. You can find the path to your workspace in Preferences -> General -> Workspace.

![2023-03-03 12_44_14-Preferences](https://user-images.githubusercontent.com/45152336/222687918-fc41d524-907d-46dc-afd0-6a8c70dde23f.png)

**Reply by sstanich** [reply]
Hi @LonwoLonwo the problem is that I can not open the program, only show this windows and close
![image](https://user-images.githubusercontent.com/74737470/222699031-a1bd06d6-0a9b-4f9d-a2c2-9dba20da5df3.png)

Dbeaver I have installed by windows store so the %APPDATA% dir does not work to get logs

**Reply by LonwoLonwo** [reply]
We have this issue, but we do not know what we can do... 
https://github.com/dbeaver/dbeaver/issues/17257

**Reply by sstanich** [reply]
Thanks @LonwoLonwo I will be checking if the issue is fixed
